src/answers-search-bar.js

```diff
@@ -1,10 +1,12 @@
 /** @module */
 
 import Core from './core/core';
+import cssVars from 'css-vars-ponyfill';
```

Comment:
would it be worth exploring a way to only load cssVars in the legacy bundles?

Reply:
We could make another entry point for the legacy search bar that has this import and the ponyfill support. It's difficult to guard `import` statements with conditionals, so that may be the best option. FWIW, the inclusion of this didn't seem to impact the performance of a page using the modern bundle. I was still seeing a score of 100. The size of the minified modern bundle wasn't impacted much by this addition.

Reply:
sure I'm fine leaving it out then, this is already a significant improvement to what we had before